docs/modules/ROOT/pages/how-tos/testing/integration/choosing-contexts.adoc

```diff
@@ -4,14 +4,15 @@ Integration test scenarios can be configured to run in only certain cases (refer
 Examples include running the integration test pipelines only in case of Pull Requests, or running them only for builds of specific components.
 The full list of supported contexts is:
 
-* “application”  - runs the integration test in all cases - this is the default state
-* “component” - runs the integration tests only in case of component builds
+* "application"  - runs the integration test in all cases - this is the default state
+* "component" - runs the integration tests only in case of component builds
 ** This context is run for builds of all components, but not for special cases like manually created Snapshots
 * “component_COMPONENT” - runs the integration test only for a build of a specific component
 ** For a component  `sample-component`, this context would need to be `component_sample-component`
-* "pull_request” - runs the integration test in case of the Snapshot being created for a `pull request` event
-* “push” - runs the integration test in case of the Snapshot being created for a `push` event
-* "override” - runs the integration test for an `override`  Snapshot
+* "pull_request" - runs the integration test in case of the Snapshot being created for a `pull request` event
+* "push" - runs the integration test in case of the Snapshot being created for a `push` event
+* "override" - runs the integration test for an `override` Snapshot
+* "group" - runs the integration test for a `group` Snapshot
 ** See more in xref:how-tos/testing/integration/override-snapshots.adoc[Creating an override snapshot]
```

Comment:
I think this existing sub-bullet point should be under the "override" context.
That said, a similar, new sub-bullet point pointing to the new group-snapshots page could be helpful here.

Reply:
Yes. I didn't notice it.....

Reply:
Updated. Thanks!